pkg/proxy/proxier.go

```diff
@@ -404,6 +466,14 @@ func iptablesInit(ipt iptables.Interface) error {
 	if _, err := ipt.EnsureChain(iptables.TableNAT, iptablesContainerPortalChain); err != nil {
 		return err
 	}
+
+	// TODO (comment from thockin): This is a little scary - if somehow these rules get out of order, this will consume
```

Comment:
I played.

`-m addrtype --dst-type LOCAL` is what we want *except* that it is broken (by intent without foresight to our usecase) on at least GCE.  Specifically, GCE machines have a daemon which learns what external IPs are forwarded to that machine, and configure a local route for that IP, making a match for  `--dst-type LOCAL` when we don't want it to.

Removing the route gives correct behavior until the daemon recreates it.

Killing the daemon is an option, but means that any non-kubernetes use of the machine with external IP will be broken.

I think the simplest thing we can do for now is to:

1) leave a TODO to enable `-m addrtype --dst-type LOCAL` with the above comments about GCE
2) make sure this jump happens last, comment it carefully
3) add a `-m comment --comment "handle service NodePorts; NOTE: this must be the last rule in the chain"` to this jump rule

What think?